c2tc-mobile/screens/LiveLocation.js

```diff
@@ -109,11 +111,7 @@ class LiveLocation extends React.Component {
             />
           ))}
         </MapView>
-        <Button
-          onPress={this._onPressToggleLayers}
-          title="Toggle Police"
-          color={policeColor}
-        />
+        <PhoneButton type="Call Neeraj" ref="button" number="6509069888" />
```

Comment:
For master I don't think you should add phonbutton in live location until daniel merges his code so maybe remove this part for now :)) Also I would check with philip and annie to make sure that your button looks similar to their designs :)))))